website/package.json

```diff
@@ -12,12 +12,12 @@
     "clear": "docusaurus clear"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-alpha.69",
-    "@docusaurus/preset-classic": "^2.0.0-alpha.69",
+    "@docusaurus/core": "2.0.0-alpha.75",
```

Comment:
This seems to be really strict. Is taking minor release w/ `^` bad?

Reply:
Not bad, it's just that after my tests using canary `2.0.0-alpha.hash` somehow using ^ did not make yarn download alpha75 and the lockfile stayed on the canary version (probably because my version name is bad)

I can revert to ^ if you want but keep in mind we force users to have all core package versions aligned. 
We'll add a "docusaurus upgrade" cli anyway so it does not matter much, up to you

Reply:
Thanks. No need to revert. We can always change that if we really needed.